playbooks/appsemblerPlaybooks/eucalyptus_pro_server.yml

```diff
@@ -0,0 +1,84 @@
+---
+
+# Appsembler pro tier deployment
+# Docs: TO-DO
+
+- name: Configure mongo
+  hosts: "mongo-server"
+  sudo: True
+  gather_facts: True
+  vars:
+    BACKUPS_MONGO: yes
+    BACKUPS_MYSQL: no
+    COMMON_ENABLE_BACKUPS: True
+  roles:
+    - { role: swapfile, SWAPFILE_SIZE: "2GB" }
+    - sudo
+    - edxapp_common
+    - mongo
+    - oraclejdk
+    - elasticsearch
+    - memcached
+    - backups
+    - role: stackdriver
+      when: "stackdriver_api_key != ''"
+
+
+- name: Configure stateless edxapp server
+  hosts: "edxapp-server"
+  sudo: True
+  gather_facts: True
+  vars:
+    migrate_db: "yes"
+    openid_workaround: True
+  roles:
+    - { role: swapfile, SWAPFILE_SIZE: "4GB" }
+    - sudo
+    - edxapp_common
+    - mysql_init
+    - role: nginx
+      nginx_sites:
+      - cms
+      - lms
+      - forum
+      nginx_default_sites:
+      - lms
+    - { role: 'edxapp', celery_worker: True }
+    - edxapp
+    - analytics_api
+    - edx_notes_api
+    - demo
+    - oauth_client_setup
+    - forum
+    - edx_ansible
```

Comment:
@tkeemon If we don't need it, will always be running ansible from our remotes, then I'd say to remove it. But I'm still new to Open edX so, I'm not sure of the conditions where we might regret not having the ability to provision locally